designsafe/static/scripts/data-depot/components/published/published-view.component.js

```diff
@@ -19,6 +19,14 @@ class PublishedViewCtrl {
         this.FileOperationService = FileOperationService;
         this.PublicationService= PublicationService;
         this.UserService = UserService;
+        this.loadingUserData = {
+            pi: true,
+            coPis: true,
+        };
+        this.authorData = {
+            pi: {},
+            coPis: null,
+        };
```

Comment:
Will it cause any issues to initialize these variables in `$onInit` instead of the constructor? If so disregard. I just wanted to mention that `this.ui` is where most of the UI params are set across different components on DS (usually) for things like "loading" etc...